small change

jiyongseong opened the pull request:
$subscriptionId = (Get-AzureRmSubscription -SubscriptionName $subscriptionName).SubscriptionId > $subscriptionId = (Get-AzureRmSubscription -SubscriptionName $subscriptionName).Id

azurecla commented:
Hi __@lauracr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

mollybostic commented:
@lauracr - Attribute values in the <tags> section all need to be lowercase. Can you change the NA values to na? Thanks!

jiyongseong committed:
small change

$subscriptionId = (Get-AzureRmSubscription -SubscriptionName $subscriptionName).SubscriptionId > $subscriptionId = (Get-AzureRmSubscription -SubscriptionName $subscriptionName).Id
